### ISSUE TITLE
Request: Preview the file without losing focus on CtrlP

### ISSUE THREAD

**Issue by hobbes3** [kien/ctrlp.vim#429]
Often times I would like to "preview" the file without actually choosing to open the file in CtrlP.

To clarify, suppose I start search and CtrlP gives me a list of 8 matched file. Since I don't exactly remember which file I want to open, I would like to see 4 of those _without losing focus or closing CtrlP_. Perhaps all 8 files were wrong and I need to modify my search.

Currently my workaround is to mark the 4 files with `<c-z>` and open them all in new tabs with `<c-t>`. But since I actually opened files, the CtrlP window will close.

**TL;DR version:** I would like CtrlP to implement the "show-results-as-you-highlight-files" feature (well it doesn't have to be on highlight; it could be on a keypress).

Thanks!
